src/pages/portfolio/index.tsx

```diff
@@ -1,11 +1,49 @@
 'use client';
 
+import React from 'react';
+import { XCircle } from 'lucide-react';
+import { Button } from '@penumbra-zone/ui/Button';
 import { Text } from '@penumbra-zone/ui/Text';
+import { Density } from '@penumbra-zone/ui/Density';
+import mobile from 'is-mobile';
 
 export const PortfolioPage = () => {
-  return (
-    <section>
-      <Text h2>Hi!</Text>
+  const isOnMobile = mobile();
+
+  return isOnMobile ? (
+    <section className='absolute inset-0 flex flex-col items-center justify-between p-4 gap-3 border-t  border-neutral-800'>
```

Comment:
issue (ui): the section renders as follows on my macbook, not matching the designs

<img width="220" alt="image" src="https://github.com/user-attachments/assets/a7e982e3-8595-4fb8-8051-5936d1ca963d">

Reply:
huh, weird. will fix.